source/BoardingPanel.cpp

```diff
@@ -544,11 +544,15 @@ BoardingPanel::Plunder::Plunder(const string &commodity, int count, int unitValu
 
 
 // Constructor (outfit installed in the victim ship or transported as cargo).
-BoardingPanel::Plunder::Plunder(const Outfit *outfit, int count)
+BoardingPanel::Plunder::Plunder(const BoardingPanel *boardingPanel, const PlayerInfo &player,
```

Comment:
```suggestion
BoardingPanel::Plunder::Plunder(const PlayerInfo &player,
```

I might overlook something, but I do not see you using this.
I also do not like to give the boardingpanel to one of it sub classes

Reply:
I dont' remember even adding it, thanks